### ISSUE TITLE
New filter for NO_METADATA

### ISSUE THREAD

**Issue by verbal666** [JosefNemec/Playnite#3623]
### Feature description

I would like to know, since i can't find now in last build, if there is the possibility to add a new filter to list ALL records without ANY metadata downloaded,

![immagine](https://github.com/JosefNemec/Playnite/assets/12701960/d3f0aa1e-f637-470c-9a9f-c345fe06e119)
![filters](https://github.com/JosefNemec/Playnite/assets/12701960/5fa9bb59-0948-45e3-8a6a-f44aa543f9c6)

In this way we can fix the record(s), manually or not, having the full list on display.

Thanks.

PS. Playnite is a wonderful piece of software 👏👏👏👏👏💪❤

**Comment by jedinjapan** [comment]
When updating the library, it would be incredibly time-saving to be able to filter by entries missing media (cover/background/icon) etc, and trigger a metadata update on only the entries with items missing. A "No Metadata" or "No Covers/Media" entry would be useful.

**Comment by scyto** [comment]
agree,  one step further filter by any metadata, for example i have ton of titles with no cover, manual search and pick works just fine (the data is there, not sure why it doesn't match) being able to show all titles without cover would speed up my workflow looking through over 3k titles of which maybe 5% don't have cover metadata,
